.github/workflows/haskell-ci.yml

```diff
@@ -14,8 +16,17 @@
 #
 name: Haskell-CI
 on:
-  - push
-  - pull_request
+  push: 
+    paths:
+      - '**.hs'
+      - '**.cabal'
+      - 'stack.yaml'
```

Comment:
Should it also keep `**.sw` and `cabal.project` ? And what about `**.yaml` for static tool configs (e.g. fourmolu.yaml) and entities if they are used in test?

Reply:
Or is there a `irrelevant-path` option we could set to `**.md` instead?

Reply:
There is, but I prefer adding .sw and cabal as I plan to add editor support which would waste Haskell CI.

Reply:
Not sure about .yaml, maybe “data/**.yaml” and “*.yaml” but not “**.yaml” would work?